src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +15,55 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
-        return null;
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert entity " + entity, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.get(Comment.class, id);
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get by id " + id, e);
```

Comment:
Not informative exception. Check your exceptions, and make them more informative
```suggestion
            throw new RuntimeException("Can't get comment by id " + id, e);
```